src/main/java/Tarnia.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Scanner;
+
+public class Tarnia {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm 𝓣𝓪𝓻𝓷𝓲𝓪 💔");
+        System.out.println("What can I do for you?");
+
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int counter = 0;
```

Comment:
Variable names should reveal intent. 
```suggestion
        int taskCounter = 0;
```